TaskManagement/src/TaskManagment/classes/Constants.java

```diff
@@ -0,0 +1,32 @@
+package TaskManagerProject;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+public class Constants {
+	public static long userID = -1; // Variable to store the user ID
+	public static long TaskID = 0; // Variable to store the task ID
+
+	public static String tabStyle = "-fx-font-weight: bold; -fx-background-color: purple; -fx-font-size: 16px; -fx-padding: 7px 7px;";

```

Comment:
The best practice in Java is to name constants with All uppercase letters separated by underscoe. For example:
`TAB_STYLE`